src/core/Akka/Actor/Props.cs

```diff
@@ -32,12 +30,12 @@ namespace Akka.Actor
     /// </code>
     ///     </example>
     /// </summary>
-    public class Props : IEquatable<Props>, ISurrogated
+    public sealed class Props : IEquatable<Props>, ISurrogated
```

Comment:
I'm not sure how I feel about this, in terms of keeping this class open for extension. I checked the Phobos source to see if we subclass `Props` anywhere there and we do not (we do subclass `Deploy`, one of the sub-components of `Props`) so if that doesn't need to extend `Props` I can't imagine many other plugins that do...